dev/beaker/clone_gatling-puppet-load-test.rb

```diff
@@ -0,0 +1,9 @@
+test_name "Clone the gatling-puppet-load-test repo on the target machine"
+
+repo = "gatling-puppet-load-test"
+repo_url = "https://github.com/puppetlabs/#{repo}.git"
+
+hosts.each do |host|
```

Comment:
But to answer your second question, it should work fine on multiple hosts, as far as I can tell. It would just create two development boxes with jenkins etc.